todo/src/App.css

```diff
@@ -1,38 +1,93 @@
+html {
```

Comment:
Usually for larger applications in React we separate our CSS based on the Component